src/roman_datamodels/maker_utils/_ref_files.py

```diff
@@ -128,6 +234,37 @@ def mk_distortion(*, filepath=None, **kwargs):
     return save_node(distortionref, filepath=filepath)
 
 
+def mk_epsf(*, shape=(3, 9, 361, 361), filepath=None, **kwargs):
+    """
+    Create a dummy ePSF instance (or file) with arrays and valid values
+    for attributes required by the schema.
+
+    Parameters
+    ----------
+    shape
+        (optional, keyword-only) Shape of arrays in the model.
+        If shape is greater than 1D, the first dimension is used.
+
+    filepath
+        (optional, keyword-only) File name and path to write model to.
+
+    Returns
+    -------
+    roman_datamodels.stnode.EpsfRef
+    """
+    if len(shape) != 4:
+        shape = (3, 9, 361, 361)
+        warnings.warn("Input shape must be 4D. Defaulting to (3, 9, 361, 361)")
+
+    epsfref = stnode.EpsfRef()
+    epsfref["meta"] = mk_ref_epsf_meta(**kwargs.get("meta", {}))
+
+    epsfref["psf"] = kwargs.get("psf", np.zeros(shape, dtype=np.float32))
+    epsfref["extended_psf"] = kwargs.get("extended_psf", np.zeros(shape[2:], dtype=np.float32))
```

Comment:
I left most of the relevant comments in the rad PR; this all looks good.  But one dumb comment here: in the actual files, is the extended_psf shape actually the same as in the psf stamps, or significantly larger?  I hope larger, even if we don't do that for the dummy epsf objects?

Reply:
The one sample file I have does not include the `extended_psf` (it is an optional array keyword). By default, the maker utility just uses a dummy shape and is not representative of reality.